config/initializers/assets.rb

```diff
@@ -1,10 +1,7 @@
 Rails.application.config.assets.precompile += %w(
-  miq_browser_detect.js
   jquery-1.8/jquery.js jquery_overrides.js
   miq_policy/import.js
   novnc-rails.js miq_novnc.js
   spice-html5.js spiceHTML5/spicearraybuffer.js
-
-  xml_display.css
-  container_topology.css service_dialogs.css import_policy.css vmrc.css
```

Comment:
@skateman I looked at `container_topology.css` and was concerned by the rules that were too generic [here](https://github.com/ManageIQ/manageiq/blob/33b85496d8cab1317ffeb5c8daebb88cdd9d2372/app/assets/stylesheets/container_topology.css#L8-L64) could you verify that those are not going to conflict being in `application.css`? (separate PR)

I did not look into the other css files.